app/mission-patches.md

```diff
@@ -151,6 +151,19 @@ We chose the plaid shirt and mug due to the fact that several team members share
 
 * * *
 
+## Get School Experience
```

Comment:
```suggestion
## Get school experience
```

Reply:
You patch has `Get school experience` so I thought the section heading should match.